### DISCUSSION TITLE
Projetos pequenos ou grandes, qual é o melhor para praticar?

### DISCUSSION THREAD

**Discussion by RicardoBaltazar** [frontendbr/forum#2229] [Perguntas e Respostas]
Na opinião de vocês, qual forma é melhor para estudar e praticar algo que acabamos não vendo no dia a dia do trabalho, criando pequenos projetos que focam em uma funcionalidade especifica e que são mais rápidos, ou criando projetos maiores com mais profundidade na tecnologia escolhida, mas que levam mais tempo para serem concluídos?

**Comment by ninetails** [comment]
É bem subjetivo. Depende do que você está estudando, de como você estuda e encara problemas. Mas mesmo na pergunta foi pra ser respondido com algo subjetivo, perguntando a opinião... Bom, a minha é:

Ainda acho que tempo é escasso, e se você não tá aprendendo algo do zero como uma nova linguagem, um novo framework... for apenas uma biblioteca, nem tem porquê despender tempo se você já souber o redor, a não ser em casos que a prova de conceito for baseada em algo maior como arquiteturas ou um projeto de curso que esteja fazendo.

**Comment by Kaedh** [comment]
concordo com o comentário do @ninetails , no meu caso sempre que estou aprendendo sobre um assunto muito especifico, busco implementar o menor projeto possível com aquilo, para ver com clareza o que eu aprendi, nessa hora percebo que muito do que eu achava que sabia vendo a teoria na realidade não sabia tanto, porém não abro mão de estar sempre criando algum projeto maior, pois sinto que sem isso eu sei muitas "peças" mas não sei junta-las pra criar algo prático. 
Mas é aquilo que ele falou vai muito de como você absorve melhor os conteúdos.
